pkgs/development/interpreters/boron/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchgit
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "boron";
+  version = "2.0.8";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/urlan/boron/code";
+    rev = "v${version}";
+    sha256 = "sha256-RpnMs9DqcbK/gLDH5x3/DvxodJb4tR76Yk8SbYtIkV4=";
+  };
+
+  # this is not a standard Autotools-like `configure` script
+  dontAddPrefix = true;
+
+  preConfigure = ''
+    patchShebangs configure
+  '';
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  buildInputs = [
+    zlib
+  ];
+
+  installTargets = [ "install install-dev" ];
```

Comment:
```suggestion
  installTargets = [ "install" "install-dev" ];
```

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/180865/commits/c6e4325e972fd53fc263918471f8be3071a7dd8d